robot/Crisis-Management/tests/browser/create_contact.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
```

Comment:
Not sure how this got in here but it can be deleted

Reply:
I had just checked in so the tests/browser folder does not remain empty.